packages/rest/src/rest.server.ts

```diff
@@ -937,6 +946,7 @@ export interface RestServerOptions {
   openApiSpec?: OpenApiSpecOptions;
   apiExplorer?: ApiExplorerOptions;
   sequence?: Constructor<SequenceHandler>;
+  queryParser?: qs.IParseOptions;
```

Comment:
I suggest that we extend it for all express related settings - see http://expressjs.com/en/4x/api.html#app.set.

```ts
expressSettings?: {[name: string]: any};
```

Reply:
Sound good. question though:

1. do you want to set the query parser custom function from the config?
1. do we really need support for `jsonp`, `views`, `view engine` and `view cache`?

Reply:
> 1. do you want to set the query parser custom function from the config?

IMO, it should be possible.

> 2. do we really need support for `jsonp`, `views`, `view engine` and `view cache`?

We can skip such options if they don't make sense to LB4. The other option is to take them as-is if they don't do any harm.

Reply:
Second try ... I think it's actually fine to leave it up to the app developper to pass whatever he/she wants as settings. Is it more inline with what you had in mind ?

*note: will update the doc once you're ok with the implementation*